docs/releasing.md

```diff
@@ -1,68 +1,88 @@
-    ```shell
-    git checkout -b release-$(echo ${RELEASE_VERSION} | cut -d '.' -f 1-2) --no-track upstream/main
-    ```
-1. Update the version of the operator in the VERSION file:
-    ```shell
-    printf "${RELEASE_VERSION}" > VERSION
-    ```
-1. Update the release manifest:
-    ```shell
-    make release
-    ```
-1. Update the base CSV `replaces` field. This is **only** needed if the
-   previous released version `${RELEASE_PREV_VERSION}` was an official release
-   i.e. no release candidate, such that users would have the previous released
-   version installed in their cluster via OLM:
     ```shell
-    printf "${RELEASE_PREV_VERSION}" > REPLACE_VERSION
+    make bundle
     ```
-1. Update bundle:
+
+11. Commit above changes:
+
     ```shell
-    make bundle
+    git commit -am "Release ${RELEASE_VERSION}"
     ```
-1. Commit above changes:
+
+12. Push the changes in the branch to your fork:
+
     ```shell
-    git commit -m "Release ${RELEASE_VERSION}" Makefile ./deploy/ ./bundle/ ./config/manifests/bases/gatekeeper-operator.clusterserviceversion.yaml ./config/manager/kustomization.yaml
+    git push -u origin create-release-${RELEASE_VERSION}
     ```
-1. Push the changes in the branch to your fork:
+
+13. Create a PR with the above changes and merge it. If using the `gh` [GitHub CLI](https://cli.github.com/) you can
+    create the PR using:
+
     ```shell
-    git push -u origin release-${RELEASE_VERSION}
+    gh pr create --repo stolostron/gatekeeper-operator --title "Release ${RELEASE_VERSION}" --body ""
     ```
-1. Create a PR with the above changes and merge it. If using the `gh` [GitHub
-   CLI](https://cli.github.com/) you can create the PR using:
-   ```shell
-   gh pr create --repo gatekeeper/gatekeeper-operator --title "Release ${RELEASE_VERSION}" --body ""
-   ```
-1. After the PR is merged, fetch the new commits:
+
+14. After the PR is merged (and any subsequent fixes), fetch the new commits:
+
     ```shell
     git fetch --prune upstream
     ```
-1. Create and push a new release tag. This will trigger the GitHub actions
-   release workflow to build and push the release image and create a new
-   release on GitHub. Note that `upstream` is used as the remote name for this
-   repository:
+
+15. Create and push a new release tag. This will trigger the GitHub actions release workflow to build and push the
+    release image and create a new release on GitHub. Note that `upstream` is used as the remote name for this
+    repository:
+
     ```shell
-    git tag -a -m "Release ${RELEASE_VERSION}" ${RELEASE_VERSION} upstream/main
+    RELEASE_VERSION="$(cat VERSION)"
+    git tag -a ${RELEASE_VERSION} -m "${RELEASE_VERSION}" upstream/main
     git push upstream ${RELEASE_VERSION}
     ```
+
+16. Disable [fast-forwarding](https://github.com/stolostron/gatekeeper-operator/actions/workflows/fast_forward.yaml) to
+    prevent unwanted commits from going to the new release.
```

Comment:
How to disable it? Can I comment all fast_forward.yaml?

Reply:
No, you just head over to the action and click "Disable" from the menu:
<img width="375" alt="image" src="https://github.com/stolostron/gatekeeper-operator/assets/19750917/8a772e2d-f24d-4142-a332-52e5c28824c6">

Reply:
(You can see an "Enable" button at the top right now since it's currently disabled.)

Reply:
amazing good tip Thanks!!